openlogin/login-with-github/index.config.ts

```diff
@@ -0,0 +1,39 @@
+import path from "path";
+import {
+  PlaywrightTestConfig,
+  PlaywrightWorkerOptions,
+} from "@playwright/test";
+import { TestArgs } from "./index.lib";
+import indexConfig from "../../index.config";
+
+const projects: Array<
+  Pick<PlaywrightWorkerOptions, "browserName"> & Omit<TestArgs, "openloginURL">
+> = [
+  {
+    browserName: "chromium",
+    user: { email: "connie.washington.1981@gmail.com" },
+  },
+  {
+    browserName: "firefox",
+    user: { email: "connie.washington.1981@gmail.com" },
+  },
+  {
+    browserName: "webkit",
+    user: { email: "connie.washington.1981@gmail.com" },
+  },
```

Comment:
my only concern is that we're using the same account for different browsers and our tests are running in parallel

Reply:
let's create a couple more users here pls

Reply:
added different github accounts for different browsers